libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -331,6 +333,50 @@ data Api routes = Api
              '[JSON]
              (ResponsesForExistedCreated "Connection existed" "Connection was created" UserConnection)
              (ResponseForExistedCreated UserConnection),
+    listConnections ::
+      routes :- Summary "List the connections to other users."
+        :> Description "You can have no more than 1000 connections in accepted or sent state"
+        :> ZUser
+        :> "connections"
+        :> QueryParam' '[Optional, Strict, Description "User ID to start from"] "start" UserId
+        :> QueryParam' '[Optional, Strict, Description "Number of results to return (default 100, max 500)"] "size" (Range 1 500 Int32)
```

Comment:
The description says maximum is 1000, but size can only be upto 500. Which one is right?

Reply:
Also, no pagination, great!

Reply:
You can have no more than 1000 connections in general. The description is about connections in general, not about this request specifically. Arguably it belongs to the "create connections" endpoint, or to all endpoints, or to another documentation page. If it's confusing, I can remove it from this endpoint.. But otherwise, you can list up to 500 in this request, and paginate by using the start parameter. I'm not sure I follow the "no pagination, great" comment, can you explain?

Reply:
I removed the description from this endpoint as it's misleading perhaps and wasn't there before.

Reply:
> I'm not sure I follow the "no pagination, great" comment, can you explain?

I thought there was no pagination, but I see now there is `start_id`, which implies pagination. 
But if we want to limit connections to 1000, why take size at all? Why not just return upto 1000 connections every time?

Reply:
Ugh, this is  a setting, so nvm about why not return all of them. Let's just implement pagination like we did for conversations.